### PR TITLE
re-wrote the post share webhook doc

### PR DESCRIPTION
**Pull request by Christian-Wathne**
## Demo

http://void-docs.talkable.com/

## Related Stories

[](https://talkable.atlassian.net/browse/PR-1234)

### CODE REVIEW

File: source/web_hooks/post_share.rst [hunk 1]
```diff
@@ -1,24 +1,43 @@
...
-Post Share Web Hook
+Post Share Webhook
 ===================
 
-Triggered when a |advocate| person shares offer details with referred persons via
-corresponding form provided by campaign workflow.
+Talkable Post Share Webhook provides notification of a share event performed within a referral campaign workflow. Use cases for the Post Share Webhook include:
+
+  * Sending automated 'Thank You' email to an Advocate for performing a share
+  * Any event that should be triggered when a user shares
```
[the remark this answers is not in the document]
Comment (Christian-Wathne) [reply source/web_hooks/post_share.rst]: Thanks @bogdan ; added

**Comment by Christian-Wathne** [comment]
Hey @zhuravel ; is there anything else I have to do on this?

**Comment by zhuravel** [comment]
@Christian-Wathne No.
I will review and release this once I have some time on my hand.
Sorry it takes so long.

**Comment by Christian-Wathne** [comment]
Awesome, thanks @zhuravel ; no hurry, I just wanted to be sure nothing was waiting on me

**Comment by zhuravel** [comment]
@Christian-Wathne Please don’t merge Pull Requests that haven’t yet been reviewed.

**Comment by Christian-Wathne** [comment]
Tjhanks @zhuravel ; will be sure in the future!

**Pull request merged**